docs/literate/src/files/index.jl

```diff
@@ -0,0 +1,10 @@
+# # Tutorials for Trixi.jl
+
+# This repository contains a tutorial section for [Trixi.jl](https://github.com/trixi-framework/Trixi.jl),
+# with interactive step-by-step explanations via [Binder](https://mybinder.org).
+
+#md # Right now, you are using the classic documentation. The corresponding notebooks can be viewed in
+#md # [nbviewer](https://nbviewer.jupyter.org/) and opened in [Binder](https://mybinder.org/) via this
+#md # icon ![](https://mybinder.org/badge_logo.svg) in the respective tutorial.
+    
+# There are tutorials for the following topics:
```

Comment:
Will there be a list online?

Reply:
Yes, a list with a short description each. I just did not copy it from TrixiTutorials until now.